symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/util/PresentationMLParser.java

```diff
@@ -0,0 +1,60 @@
+package com.symphony.bdk.core.service.message.util;
+
+import com.symphony.bdk.core.service.message.exception.PresentationMLParserException;
+
+import lombok.Generated;
+import lombok.SneakyThrows;
+import org.apiguardian.api.API;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+/**
+ * Helper class for parsing the PresentationML to text content.
+ */
+@API(status = API.Status.STABLE)
+public class PresentationMLParser {
+
+  private static final DocumentBuilder builder = initBuilder();
```

Comment:
is it thread safe?

Reply:
Apparently not https://stackoverflow.com/a/12455887 (my initial mistake)

Reply:
oh oh it looks like it is not https://stackoverflow.com/questions/12455602/is-documentbuilder-thread-safe